lib/octokit/client/reviews.rb

```diff
@@ -0,0 +1,184 @@
+module Octokit
+  class Client
+
+    # Methods for the Reviews API
+    #
+    # @see https://developer.github.com/v3/reviews/
+    module Reviews
+
+      # List reviews on a pull request
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param id [Integer] The id of the pull request
+      # @see https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
+      #
+      # @example
+      #   @client.pull_request_reviews('octokit/octokit.rb', 2)
+      #
+      # @return [Array<Sawyer::Resource>] Array of Hashes representing the reviews
+      def pull_request_reviews(repo, id, options = {})
+        options = ensure_api_media_type(:reviews, options)
+        get "#{Repository.path repo}/pulls/#{id}/reviews", options
+      end
+
+      # Get a single review
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param pull_id [Integer] The id of the pull request
+      # @param review_id [Integer] The id of the review
+      # @see https://developer.github.com/v3/pulls/reviews/#get-a-single-review
+      #
+      # @example
+      #   @client.pull_request_review('octokit/octokit.rb', 825, 6505518)
+      #
+      # @return [Sawyer::Resource] Hash representing the review
+      def pull_request_review(repo, pull_id, review_id, options = {})
+        options = ensure_api_media_type(:reviews, options)
+        get "#{Repository.path repo}/pulls/#{pull_id}/reviews/#{review_id}", options
+      end
+
+      # Delete a pending review
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param pr_id [Integer] The id of the pull request
+      # @param review_id [Integer] The id of the review
+      # @see https://developer.github.com/v3/pulls/reviews/#delete-a-pending-review
+      #
+      # @example
+      #   @client.delete_pull_request_review('octokit/octokit.rb', 825, 6505518)
+      #
+      # @return [Boolean] True if deleted, false otherwise
+      def delete_pull_request_review(repo, pull_id, review_id, options = {})
+        options = ensure_api_media_type(:reviews, options)
+        boolean_from_response(
```

Comment:
Since the API returns a payload (https://developer.github.com/v3/pulls/reviews/#delete-a-pending-review) this shouldn't be a `boolean_from_response`. Those should be reserved for requests that respond with a `204`.

See comment in `/lib/octokit/connection.rb`